src/raftstore/store/worker/apply.rs

```diff
@@ -375,11 +382,20 @@ impl ApplyDelegate {
                         panic!("{} failed to write to engine, error: {:?}", self.tag, e)
                     });
 
+                if !apply_ctx.raft_wb_ref().is_empty() {
+                    self.raft_engine
+                        .write(apply_ctx.raft_wb.take().unwrap())
+                        .unwrap_or_else(|e| {
+                            panic!("{} failed to write to engine, error: {:?}", self.tag, e)
+                        });
+                }
+
                 // call callback
                 for (cb, resp) in apply_ctx.cbs.drain(..) {
                     cb(resp);
                 }
                 apply_ctx.wb = Some(WriteBatch::with_capacity(DEFAULT_APPLY_WB_SIZE));
+                apply_ctx.raft_wb = Some(WriteBatch::with_capacity(DEFAULT_APPLY_WB_SIZE));
```

Comment:
In most cases, we don't need to write raft rocksdb, only for change peer and split, so here we don't need to create the writebatch with capacity every time.